utils/remote-ext-tests/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "remote-ext-tests"
+version = "0.9.8"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+
+[dependencies]
+polkadot-runtime = { version = "0.9.8", path = "../../runtime/polkadot", optional = true }
+kusama-runtime = { version = "0.9.8", path = "../../runtime/kusama", optional = true }
+
+pallet-staking = { git = "https://github.com/paritytech/substrate", branch = "master", features = ["runtime-benchmarks"] }
```

Comment:
Using the runtime-benchmarks feature because it triggers compilation of the `node.voter()` fn